client/android/app/src/main/java/com/snapscreen/aerogear/AerogearModule.java

```diff
@@ -19,12 +23,13 @@
     private final String VARIANT_ID       = "3e2c6bd7-cd69-49eb-bd9e-a9360e79f1a9";
     private final String SECRET           = "15448edc-8896-4163-aca6-cbf783f9165b";
```

Comment:
Short answer yes.  

Ideally we should use the push-config.json file provided by UPS.